js/ui/select_box.js

```diff
@@ -232,6 +233,7 @@ const SelectBox = DropDownList.inherit({
     _createPopup: function() {
         this.callBase();
         this._popup.$element().addClass(SELECTBOX_POPUP_CLASS);
+        this.$element().find(`.${OVERLAY_CONTENT_CLASS}`).attr('tabindex', -1);
```

Comment:
```suggestion
```

Let's try to extend Popup content:

```js
_popupConfig: function() {
        return extend(this.callBase(), {
            tabIndex: -1
        });
    },
```

Reply:
It does not work, because popupConfig is extended width dropDownOptions cache on the dropDownEditor level

Reply:
So `this._popup.option('tabIndex', -1);`

Reply:
fixed